dev/encoding/vqgan-jax-encoding-yfcc100m-streaming.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>I actually like to have separate functions in the notebook as we can explore what's happening. Each dataset may be a bit different.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/borisdayma/dalle-mini/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Do you mean for the final encoding function? We can try to reuse some of the previous work, I'll try to set it up.</p>

Reply:
Oh it's true that there's actually 2 splits in this case so may not be a bad idea.